error-handler.js

```diff
@@ -269,6 +277,9 @@ createHandler = function createHandler(options) {
       return resumeOrClose(status);
     }
 
+    // Always set a response status.
```

Comment:
Whoa, every IE8 battered XDR GET/POST only HTTP protocol subverted bone in my body just shivered. =)
Shouldn't this library (this level, this scope) just be standards compliant? What you're describing  sounds like something beyond the scope.
If a user was interested in security, HTTPS or F5 appliance?
(Both will rely on the standards NOT being subverted by this library)